addons/l10n_id_efaktur/views/account_move_views.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_move_efaktur_form_view" model="ir.ui.view">
+            <field name="name">account.move.efaktur.form.view</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="arch" type="xml">
+                <field name="partner_id" position="after">
+                    <field name="l10n_id_need_kode_transaksi" invisible="1"/>
+                    <field name="country_code" invisible="1"/>
+                    <field name="l10n_id_kode_transaksi" attrs="{'invisible': ['|', ('country_code', '!=', 'ID'), ('l10n_id_need_kode_transaksi', '=', False)], 'required': [('l10n_id_need_kode_transaksi', '=', True)]}"/>
+                    <field name="l10n_id_replace_invoice_id" attrs="{'invisible': [('country_code', '!=', 'ID')], 'readonly': [('state', '!=','draft')]}" options="{'m2o_dialog': False, 'no_create': True}"/>
+                </field>
+                <field name="type" position="after">
+                    <span style="font-size: 11px;" attrs="{'invisible': ['|', ('country_code', '!=', 'ID'), '|', ('l10n_id_date_validated', '!=', False), ('l10n_id_tax_number', '=', False)]}" class="label label-danger">
+                        <strong>e-Faktur: Not yet Uploaded</strong>
+                    </span>
+                    <span style="font-size: 11px;" attrs="{'invisible': ['|', ('country_code', '!=', 'ID'), ('l10n_id_date_validated', '=', False)]}" class="label label-success">
+                        <strong>e-Faktur: Uploaded</strong>
+                    </span>
+                </field>
+                <button name="button_draft" position="after">
+                    <button string='E-faktur' name="download_efaktur" type="object" attrs="{'invisible':['|', ('country_code', '!=', 'ID'), '|', ('l10n_id_csv_created','=',True), ('state', '=', 'draft')]}"/>
+                    <button name="reset_efaktur" string="Reset E-Faktur" type="object" attrs="{'invisible':['|', ('country_code', '!=', 'ID'), '|','|',('l10n_id_tax_number','=',False),('state','!=','cancel'),('l10n_id_date_validated','!=',False)]}"/>
```

Comment:
If `l10n_id_date_validate != False`, it means it has been validated and uploaded. 
In that case, the button is invisible.

If `l10n_id_date_validate == False`, it means we didn't upload yet to the government.
In that case, the button is visible and we can recycle the number without problems, because we are allowed to do it by both Odoo and the tax office.

Reply:
@william-andre Ok, so you can not change it when you sent it to the government (if you do that, you need a replace_invoice_id).  Now I get it.